src/components/Link/Link.stories.tsx

```diff
@@ -113,14 +92,48 @@ const NextJsLink = (props: any) => (
     </div>
 );
 
-export const LinkWithNextJSRouter = () => (
+export const linkWithOtherRouters = () => (
```

Comment:
no knobs on these guys?

Reply:
My imagination failed me for these 'uns—what do you think would be a helpful knob?